source/ggplotd/geom.d

```diff
@@ -551,25 +585,29 @@ auto geomAxis(AES)(AES aesRaw, double tickLength, string label)
     double[] langles;
     string[] lbls;
 
-    auto merged = DefaultValues.mergeRange(aesRaw);
```

Comment:
Did you test whether this is a big performance improvement? mergeRange should be mostly compile time, and not really cause a slow down (I'd have thought)